app/classifier/tasks/textFromSubject/editor.jsx

```diff
@@ -0,0 +1,97 @@
+import { MarkdownEditor, MarkdownHelp } from 'markdownz';
+import PropTypes from 'prop-types';
+import React from 'react';
+import AutoSave from '../../../components/auto-save';
+import alert from '../../../lib/alert';
+import handleInputChange from '../../../lib/handle-input-change';
+import NextTaskSelector from '../next-task-selector';
+
+export default function TextFromSubjectEditor({
+  task,
+  taskPrefix,
+  workflow
+}) {
+  const handleChange = handleInputChange.bind(workflow);
+
+  return (
+    <div className="text-editor">
+      <section>
+        <div>
+          <AutoSave resource={workflow}>
+            <span className="form-label">Main text</span>
+            <br />
+            <textarea
```

Comment:
```suggestion
            <textarea
              id={`${taskPrefix}-instruction`}
```
All form inputs must be labelled.